h/views/admin/groups.py

```diff
@@ -27,18 +26,15 @@
 )
 @paginator.paginate_query
```

Comment:
The paginator decorator makes stuff just a little fiddly to work with here. It requires an SA Query object (versus the results of a query).